test/integration/GuardedStream.test.ts

```diff
@@ -4,7 +4,7 @@ import {
   readableToString,
   ChainedConverter,
   guardedStreamFrom,
-  RdfToQuadConverter, BasicRepresentation, getLoggerFor,
+  RdfToQuadConverter, BasicRepresentation, getLoggerFor, INTERNAL_QUADS,
```

Comment:
weird formatting to have this on one line but not the rest

Reply:
Really need to find a way to make my IDE stop doing that.